assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -84,7 +97,6 @@ function ContributionAmount(props: PropTypes) {
               min="1"
```

Comment:
Could make sense to hook up the other amount field in this PR as well since it's so closely related to the rest

Reply:
Yeah indeed. I will abstract this in the contributions metadata as I think it is region-specific, and then can use it in `otherAmount` too.